_posts/2015-12-01-minutes-offline-case-study.md

```diff
@@ -0,0 +1,101 @@
+---
+
+title: 'A Hoodie Case Study: How minutes.io does offline'
+author: Alex Feyerke
+layout: blog-post
+comments: true
+
+---
+
+minutes.io is a so-called [offline-first](http://offlinefirst.org) web app: after the first visit, it considers a working network connection an optional enhancement, and tries to enable as much functionality as possible without it. It’s been around for about five years, and none of the tech used is particularly new or spectacular, but it’s robust and does exactly what it’s supposed to. So here’s how we did it.
```

Comment:
I'd get rid of "a so-called" and replace it with "an". Purely because when I say "so-called" it's usually meant as a negative term.

> adjective
synonyms:	inappropriately named; More
used to express one's view that such a name or term is inappropriate.
"she could trust him more than any of her so-called friends"

^ from Google